src/quacc/calculators/mrcc/mrcc.py

```diff
@@ -0,0 +1,147 @@
+from __future__ import annotations
+
+import re
+
+from ase.calculators.genericfileio import (
+    BaseProfile,
+    CalculatorTemplate,
+    GenericFileIOCalculator,
+)
+
+import quacc.calculators.mrcc.io as io
+
+
+def get_version_from_mrcc_header(mrcc_header):
+    match = re.search(r"Release date: (.*)$", mrcc_header, re.M)
+    return match.group(1)
```

Comment:
We don't want this function to be user-facing in the docs, so please prepend it with `_`. I know ASE did not do that.